packages/core/lib/debug/printer.js

```diff
@@ -86,6 +88,16 @@ class DebugPrinter {
     );
   }
 
+  warnIfNoSteps() {
+    if (this.session.view(trace.steps).length === 0) {
+      this.config.logger.log(
+        `${colors.bold(
+          "Warning:"
+        )} this transaction has no trace steps. This may happen if you are attempting to debug a transaction sent to an externally-owned accounts, or if the node you are connecting to failed to produce a trace for some reason. Please check your configuration and try again.`
```

Comment:
Grammar error or typo here:

```suggestion
        )} this transaction has no trace steps. This may happen if you are attempting to debug a transaction sent between externally-owned accounts, or if the node you are connecting to failed to produce a trace for some reason. Please check your configuration and try again.`
```